sci-electronics/kicad/kicad-9999.ebuild

```diff
@@ -42,6 +60,7 @@ COMMON_DEPEND="
 	dev-db/unixODBC
 	dev-libs/boost:=[context,nls]
 	dev-libs/libgit2:=
+	dev-libs/protobuf
```

Comment:
If protobuf is linked against, this needs to be := instead (and a revbump, so git mv the 9.0.0 ebuild to -r1 if so).

If just `protoc` is used, then it needs to be BDEPEND-only, though.